.template/addons/github/.github/workflows/test.yml.tt

```diff
@@ -144,7 +144,7 @@ jobs:
     needs: unit_tests
     runs-on: ubuntu-latest
     timeout-minutes: 5
-    if: github.event_name == 'pull_request'
+    if: github.event_name == 'pull_request' && !contains(join(github.event.pull_request.labels.*.name, ','), 'wip') && !contains(github.event.pull_request.title, '[wip]') && !github.event.pull_request.draft
```

Comment:
I have one more question - could you clarify that if we add the label `WIP` **after** creating the PR, does this workflow trigger successfully?

Reply:
Umm, no. This check triggers only when creating the PR. 

To trigger the workflow each time a label or title has been changed or PR status changed to/from draft we will have to add change shown below.

Should I add these changes? 


```
on:
  pull_request:
    types: [opened, edited, labeled, unlabeled, converted_to_draft, ready_for_review]
  push:
    branches:
      - develop
      - main
```

Reply:
IMO, I think it's good idea to add it @mosharaf13. 😉

Reply:
Thanks 👍🏼 . Added in b925ebc529f1ade2688f5ad2e730e94ced7a42b9